vendor/k8s.io/kubernetes/test/e2e/generated/bindata.go

```diff
@@ -2278,7 +2278,7 @@ func testE2eTestingManifestsStorageCsiGcePdController_ssYaml() (*asset, error) {
testE2eTestingManifestsStorageCsiGcePdCsiControllerRbacYaml = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\xcc\x57\xc1\x8e\xe2\x38\x10\xbd\xe7\x2b\x4a\xf4\x65\x57\x22\x44\xbd\xa7\x15\xb7\x5d\x0e\x7b\x45\xcd\xaa\x2f\xa3\x39\x38\x4e\x75\xf0\xe0\xd8\x96\xab\x9c\x6e\xfa\xeb\x47\x0e\x01\x85\x81\x81\x34\x10\xba\xb9\x10\x45\x65\xbf\x7a\xe5\x7a\xcf\x95\x87\xf8\x83\x99\x35\xec\xad\xd6\xe8\x61\x81\xbe\x56\x12\xe1\x1f\x29\x6d\x30\x3c\x86\x27\xab\x91\x36\x7f\xb9\x32\x85\x32\x25\x25\xc2\xa9\x67\xf4\xa4\xac\x99\x42\xfd\x98\xac\x94\x29\xa6\xdb\x95\xed\xc2\xa4\x42\x16\x85\x60\x31\x4d\x00\x8c\xa8\x70\x0a\x92\x54\x5a\x4a\x4c\x5d\x91\xca\x1d\x60\x4a\x22\x49\xd2\x34\x4d\x1e\xe0\xcd\xe3\xcb\x14\x96\xcc\x8e\xa6\x59\x56\x2a\x5e\x86\x7c\x22\x6d\x95\xad\x42\x8e\xde\x20\x23\xa5\x92\x54\x86\x6f\x8c\xde\x08\x9d\x3a\x6f\x6b\x15\xb3\x40\x9f\xe5\xda\xe6\x59\x25\x88\xd1\x67\x05\x3a\x6d\xd7\x9d\x65\x99\xcf\x85\x9c\xac\x45\xa5\xdb\x5c\x67\x3a\xc4\xc8\x48\x6a\x8f\x4c\x13\x27\x02\x2f\xad\x57\xef\x82\x95\x35\x93\xd5\xdf\x34\x51\x36\xab\x1f\x4f\x33\xea\xe4\x92\xfa\xb8\xad\x0f\x1a\x29\x86\xa6\x20\x9c\xfa\xcf\xdb\xe0\x68\x0a\xdf\x46\xa3\xef\x09\x00\x80\x47\xb2\xc1\x4b\x6c\xde\xb9\x88\x4f\x8c\x86\x6b\xab\x43\x85\xd4\x06\xd5\xe8\xf3\x26\xa0\x44\x1e\x8d\x61\xa4\x15\x35\xff\xaf\x82\xe5\x32\x3e\x48\x8f\x82\x31\x3e\x15\xa8\x91\xb1\x59\x77\x01\xa0\xd4\x42\x55\xbd\x51\x83\x2b\xc4\x71\x2c\x62\xeb\x45\x89\x6d\xd1\x8e\x21\xb7\x11\x52\x0b\xa2\x9e\x3c\x7b\x72\xc2\x1a\x0d\x1f\xec\x78\xa2\x64\x2d\x8d\x31\x8c\xdc\xef\x70\xce\xf3\x91\xa4\x8c\x2d\x6e\xcb\xe4\xaa\x0d\xc9\x08\x47\x4b\xcb\x93\xf3\xb9\xb7\xcd\xd6\x2e\x38\x89\x78\x4b\xa0\x28\xfe\x63\x47\xb5\x8f\xd7\x98\xc2\xa1\x5c\xff\xdd\x78\xd0\x8d\x54\xdb\xf1\xa1\xae\x80\x5b\xa3\x4b\x28\xe4\x3f\x50\x72\x2b\xe3\xa3\x36\x17\x19\x9c\xb1\xb7\x68\x07\x4f\xf8\x12\x37\x39\x74\x9f\x3e\x56\x02\xbb\xc2\x9f\xe0\x9a\x00\x5c\xea\xa3\x82\x59\xc8\xe5\x17\x30\xd1\x6d\x22\x77\x74\xd0\x1e\x26\x70\x7b\x89\x0e\xee\x2a\x7d\x25\xb9\x29\x78\x75\x4e\x8d\xa7\x0b\x76\x6f\xa5\xee\xba\xe4\x73\x65\xba\xdf\xac\xfd\x34\xba\x29\xd5\x03\x3c\x21\xa9\x77\xf4\x50\x05\x62\xc8\x11\x44\xae\x11\xd8\xc2\xab\xf5\x2b\x78\x55\xbc\x84\xf9\xf3\x8c\xc6\x30\x7f\xa6\x31\x2c\x66\x34\x19\x4c\x72\x7e\x93\xc9\xd7\x57\xdc\x15\x23\xcb\x55\x08\x19\xb1\xe0\x70\x00\x74\x21\x8b\x1b\x0c\x29\x4d\x0f\x0d\xac\xb6\x6d\x57\x7c\xae\xc2\xf6\x7a\xf3\x03\x02\xbb\xe0\x0a\xdc\xce\x26\x7c\xf6\x16\x8c\xf9\x75\xa3\x63\x2e\xe9\x2f\x2f\x87\xbd\x25\x3b\x40\x1f\x97\xed\x20\x53\xf2\xc5\x93\xe0\x75\x5f\x01\x83\x4e\xa0\xc7\x2d\xab\xfd\xcc\x1a\xa6\x1a\x6d\x56\xbd\x1d\xe7\x1e\x4e\xd0\x55\x70\xa7\xef\x3e\xd7\x18\x0e\x14\xf0\x21\x73\x70\x5e\xd5\x4a\x63\x51\x62\x01\x73\x5b\xc0\x02\x65\xf0\x8a\xd7\x30\xb7\x5a\xc9\xf5\x18\x9c\xc7\x5a\xd9\x40\x7a\x0d\x05\xbe\x28\x83\x05\xd4\x4a\xc0\x7c\xb3\xae\xc4\xe2\x7f\x24\x9e\x2f\xe6\x87\x45\xff\xe3\xcf\x41\xce\xc3\x91\x6b\x2c\xa6\x53\xb5\xc2\xab\x3a\xb2\xc7\x52\x11\x7b\xd1\xd6\xe1\x8e\xc7\x81\x7f\x61\xca\x48\x9c\xba\x5d\x59\x52\x47\xae\xe7\x51\xfc\x0c\x00\x00\xff\xff\x9f\xb0\xc3\xe5\x78\x12\x00\x00")
```

Comment:
The diff in this file wasn't part of the upstream PR, but it was created when I ran `hack/update-generated-bindata`.